aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -1172,6 +1175,10 @@ struct Pack : public KernelCache <PackKey, pack_t> {
     if (dtype == ScalarType::Half) {
       static bool fp16_pack = dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx512_core_amx_fp16;
       return fp16_pack;
+    } else if (dtype == ScalarType::BFloat16) {
+      // TODO Enable it on avx512_core_amx when bf16 perfomance is better
+      static bool bf16_pack = dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx512_core_amx_fp16;
```

Comment:
why is it good with amx_fp16 then? Could you please explain in the note?

Reply:
Enable packing on avx512_core_amx for bf16, and wait for the test results.